examples/guestbook/README.md

```diff
@@ -96,27 +101,36 @@ CONTAINER ID        IMAGE                                  COMMAND
 ### Step Two: Fire up the master service
 A Kubernetes 'service' is a named load balancer that proxies traffic to *one or more* containers. This is done using the *labels* metadata which we defined in the redis-master pod above.  As mentioned, in redis there is only one master, but we nevertheless still want to create a service for it.  Why?  Because it gives us a deterministic way to route to the single master using an elastic IP.
 
-The services in a Kubernetes cluster are discoverable inside other containers via environment variables. 
+The services in a Kubernetes cluster are discoverable inside other containers via environment variables.
 
 Services find the containers to load balance based on pod labels.
 
-The pod that you created in Step One has the label `name=redis-master`. The selector field of the service determines *which pods will receive the traffic* sent to the service, and the port and containerPort (targetPort in v1beta3 and higher APIs) information defines what port the service proxy will run at. 
+The pod that you created in Step One has the label `name=redis-master`. The selector field of the service determines *which pods will receive the traffic* sent to the service, and the port and containerPort (targetPort in v1beta3 and higher APIs) information defines what port the service proxy will run at.
```

Comment:
Or remove mention of the old fields entirely, since we're removing the old example, anyway.